testing-basics-garden-project/garden-test012.js

```diff
@@ -0,0 +1,23 @@
+import { Garden } from "../src/Garden";
+import { FruitTree } from "../src/FruitTree";
+const chai = require("chai");
+const assert = chai.assert;
+
+describe("Garden tests", function () {
+   let myGarden = new Garden();
+   let myFruitTree = new FruitTree("Strawberry");
+   it("should be able to create a Garden", function () {
+      assert.instanceOf(myGarden, Garden);
+   });
+
+   it("should be able to add fruit trees", function () {
+      myGarden.addPlant(myFruitTree);
+      assert.isAbove(myGarden.fruitTrees.length, 0);
+   })
+});
+
+/*  Congrats! You have now made two test cases!
+    Let's create another test case called "should not allow more than 5 fruit trees"
+    This will correspond to the next requirement.
+    You don't need to implement the test yet.
+*/
```

Comment:
```
/*  
  Congrats! You have now made two test cases!
  Let's create another test case called `"should not allow more than 5 fruit trees"`.
  This will correspond to the next requirement.
  You don't need to implement the test yet.
*/
```